owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -127,6 +130,7 @@ class MainFileListFragment : Fragment(),
         )
     }
     private val fileOperationsViewModel by sharedViewModel<FileOperationsViewModel>()
+    private val transfersViewModel by viewModel<TransfersViewModel>()
```

Comment:
This could not be managed with the MainFileListViewModel or by fileOperationsViewModel? I think it's a better solution to not add not view logic to the fragment.

Reply:
Technically it could, but we already have this logic in the TransfersViewModel and:

1) We would need to duplicate code (not a good practice)
2) Semantically, this is something related with transfers so we shouldn't mix it up with other ViewModels

So, taking into account we already have several ViewModels in this fragment, adding another one to prevent duplication of code, I think it's the best option